### PR TITLE
Add custom healthcheck function

### PR DESCRIPTION
**Pull request by lennym**
Makes a call to the elasticsearch API to ensure that it is available when `/healthcheck` is hit.

### CODE REVIEW

**Comment by lennym** [comment]
Waiting for https://github.com/UKHomeOffice/asl-service/pull/211

File: lib/router/search.js [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
 
-  const client = createESClient(settings.es);
-
   app.use((req, res, next) => {
-    return Promise.resolve(client)
-      .then(esClient => {
-        req.search = search(esClient);
+    return Promise.resolve(settings.esClient)
+      .then(client => {
+        req.search = search();
```
Comment (wheelsandcogs) [inline lib/router/search.js:12]: don't you still need to pass the client here?

Comment (lennym) [reply lib/router/search.js]: Yes, my bad. I thought the way it was coded was weird with the Promise wrapper, then realised after I'd changed it why it was necessary but didn't fully put it back clearly.

**Comment by lennym** [comment]
No longer WIP.

**Pull request merged**